submarine-sdk/pysubmarine/example/deepFM.json

```diff
@@ -0,0 +1,31 @@
+{
+  "input": {
+    "train_data": "./data/tr.libsvm",
```

Comment:
Thanks for your contribution! @pingsutw  It looks good to me. One minor suggestion, can we use the hyphen instead of underscore? The underscore character seems more preferred from a developer view and is not user-friendly to me.

Reply:
@tangzhankun Thanks for the review. I like hyphen as well, but I follow other model convention like [xgboost](https://xgboost.readthedocs.io/en/latest/parameter.html), [sklearn](https://scikit-learn.org/stable/auto_examples/linear_model/plot_ols.html), it looks like data scientists much prefer to underscore

Reply:
I see. Let's keep it underscore then.